### PR TITLE
Added AzureAutomation logger

### LINKED ISSUE #15: Log Harvester
Log harvester should be created that harvests logs into a storage account

### PR DESCRIPTION
**Pull request by lfshr**
Added a module for Azure Automation logging. This closes #15.

### CODE REVIEW

File: src/Logging/AzureAutomationTools.Logging.psm1
Comment (wibblemonkey) [inline src/Logging/AzureAutomationTools.Logging.psm1]: Misaligned line.

**Review by wibblemonkey — CHANGES_REQUESTED** [review]
Looks fine to me except for that one misaligned line. Good stuff.

File: src/Logging/AzureAutomationTools.Logging.psm1
Comment (lfshr) [reply src/Logging/AzureAutomationTools.Logging.psm1]: Fixed in latest commit - Formatted document.

**Review by wibblemonkey — APPROVED** [review]

**Comment by lfshr** [comment]
@stvnrs there are a few changes to your stuff here so I'll wait for your review before merging

File: src/Logging/AzureAutomationTools.Logging.psm1
Comment (stvnrs) [inline src/Logging/AzureAutomationTools.Logging.psm1:150]: This should be optional - quite dangerous as a default

File: src/Logging/AzureAutomationTools.Logging.psm1
Comment (stvnrs) [inline src/Logging/AzureAutomationTools.Logging.psm1]: Should support ShouldProcess in next version

File: src/Logging/AzureAutomationTools.Logging.psm1
Comment (stvnrs) [inline src/Logging/AzureAutomationTools.Logging.psm1:42]: Remove this param - should not login.

File: src/Logging/AzureAutomationTools.Logging.psm1
Comment (stvnrs) [inline src/Logging/AzureAutomationTools.Logging.psm1]: Remove

File: src/Logging/AzureAutomationTools.Logging.psm1
Comment (stvnrs) [inline src/Logging/AzureAutomationTools.Logging.psm1]: Needs a new line

**Review by stvnrs — CHANGES_REQUESTED** [review]
Changes required - see comments

**Review by stvnrs — APPROVED** [review]

File: src/Logging/AzureAutomationTools.Logging.psm1
Comment (lfshr) [reply src/Logging/AzureAutomationTools.Logging.psm1:150]: Hard error in commit tree. Future release may make this optional.

File: src/Logging/AzureAutomationTools.Logging.psm1
Comment (lfshr) [reply src/Logging/AzureAutomationTools.Logging.psm1:42]: Done.

File: src/Logging/AzureAutomationTools.Logging.psm1
Comment (lfshr) [reply src/Logging/AzureAutomationTools.Logging.psm1]: Done.

**Pull request merged**
